src/hotspot/os/linux/os_perf_linux.cpp

```diff
@@ -785,7 +785,7 @@ char* SystemProcessInterface::SystemProcesses::ProcessIterator::get_exe_path() {
 
   jio_snprintf(buffer, PATH_MAX, "/proc/%s/exe", _entry->d_name);
   buffer[PATH_MAX - 1] = '\0';
-  return realpath(buffer, _exePath);
+  ALLOW_C_FUNCTION(::realpath, return realpath(buffer, _exePath);)
```

Comment:
Shouldn't this be using `os::Posix::realpath`?

Reply:
Yes, I agree. Fixed.